packages/earth-admin/src/components/dropdown/Dropdown.tsx

```diff
@@ -30,8 +30,8 @@ interface IDropdownComponent {
 }
 
 const DropdownPanel: any = Keyframes.Spring({
-  close: { x: -1000, delay: 0 },
-  open: { x: 0, from: { x: 0 } },
+  close: { x: -1000, delay: 0, config: { duration: 0 } },
+  open: { x: 0, from: { x: 0 }, config: { duration: 0 } },
```

Comment:
Please remove the unnecessary animation components instead of just setting the animation to 0. 
This creates tech debt.

Reply:
We agreed to keep it this way until we select a UI framework that already has a dropdown component